src/node/.eslintrc.json

```diff
@@ -10,32 +10,32 @@
   ],
   "overrides": [],
   "parserOptions": {
-      "ecmaVersion": "latest",
-      "sourceType": "module",
-      "project": "src/node/tsconfig.json"
-    },
+    "ecmaVersion": "latest",
+    "sourceType": "module",
+    "project": "./tsconfig.json"
```

Comment:
Should be moved to another PR

Reply:
Agree: https://github.com/cloudflare/workerd/pull/2554